detection_rules/devtools.py

```diff
@@ -180,11 +177,11 @@ def license_check(ctx):
 
     failed = False
 
-    for path in glob.glob(get_path("**", "*.py"), recursive=True):
-        if path.startswith(get_path("env", "")):
+    for path in ROOT_DIR.rglob("*.py"):
+        if 'env' in path.parts:
             continue
 
-        relative_path = os.path.relpath(path)
+        relative_path = Path(os.path.relpath(path))
```

Comment:
Looking at how to replace `os.path.relpath` with pathlib's equivalent (if it has one).

> Note: Although os.path.relpath() and PurePath.relative_to() have some overlapping use-cases, their semantics differ enough to warrant not considering them equivalent.

Source: https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
This might do the trick

```
from pathlib import PurePath

PurePath(path)
```